v_4_0_0/render_asset_content_test.go

```diff
@@ -36,3 +36,27 @@ func TestRenderAssetContent(t *testing.T) {
 		assert.Equal(t, tc.expectedContent, content, "content should be equal")
 	}
 }
+
+func TestRenderFileAssetContent(t *testing.T) {
+	tests := []struct {
+		assetContent    string
+		params          FakeParams
+		expectedContent string
+	}{
+		{
+			assetContent: testTemplate,
+			params:       FakeParams{Foo: "bar"},
+			// expected base64 encoding of `foo: bar`
+			expectedContent: "Zm9vOiBiYXI=",
+		},
+	}
+
+	for _, tc := range tests {
+		content, err := RenderFileAssetContent(tc.assetContent, tc.params)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		assert.Equal(t, tc.expectedContent, content, "content should be equal")
+	}
+}
```

Comment:
It would be good to align newly written tests with what we have here https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md.